src/main/java/de/orchound/gameoflife/parsing/RleBoardParser.java

```diff
@@ -13,7 +13,7 @@
 
 	private final int extraBoardSize = 80;
 	private final int halfExtraBoardSize = extraBoardSize / 2;
-	private final Pattern dimensionPattern = Pattern.compile("x\\s?=\\s?(\\d+),\\s?y\\s=\\s?(\\d+)");
+	private final Pattern dimensionPattern = Pattern.compile("x\\s?=\\s?(\\d+),\\s?y\\s=\\s?(\\d+),\\srule\\s=\\s(B\\d+/S\\d+)");
```

Comment:
The new stuff is clearly not part of the "dimension pattern".
Add a new member `rulePattern`, then match the same line twice.